openfl/interface/aggregator.py

```diff
@@ -183,6 +189,7 @@ def certify(fqdn, silent):
 
         if confirm('Do you want to sign this certificate?'):
 
+            echo('Make sure the two hashes above are the same.')
```

Comment:
Per the requirement:

"User is expected to compare the two hashes and approve the request only if they match. It may be useful to emphasize this expectation, for example in openfl documentation and/or as part of the prompt"

At this point the decision has already been made to sign the certificate. A message prompting for verification should appear before line 190